sync-core/src/test/java/com/cloudant/sync/datastore/DatastoreSchemaTests.java

```diff
@@ -46,4 +66,174 @@ public void testSchemasSame() {
         assertThat("Schemas should be the same", schema_Alpha5, equalTo(schema_Beta5));
     }
 
+
+    @Test
+    /**
+     * Test the specific migration of a version 6 database to a version 7 database.
+     * This migration should add the attachments_key_filename table, and populate
+     * it with the existing keys from the attachments table as both key and filename.
+     *
+     * fixture/datastores-user_version6.zip contains a known database, at version 6 and
+     * with two attachments.
+     */
+    public void migrateVersion6To7() throws DatastoreNotCreatedException, ExecutionException,
```

Comment:
It doesn't affect the logic of the test but for consistency you should change 7 to 100 in a few places here